micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/ServiceRestClientConfiguration.groovy

```diff
@@ -35,6 +37,9 @@ import java.util.concurrent.ThreadFactory
 @EnableServiceDiscovery
 class ServiceRestClientConfiguration {
 
+    @Value('${microservice.restclient.connectionTimeout:-1}') int connectionTimeoutMillis
```

Comment:
Maybe we do want to have some liberal (like 1 minute) but still hard-coded timeout? Not sure...